miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/service/RunScannerClient.java

```diff
@@ -89,8 +89,8 @@ private void processResults(List<NotificationDto> results) {
       }
       for (NotificationDto dto : results) {
         try (AutoCloseable timer = saveTime.start()) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Extract this nested try block into a separate method. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1141)